docs/README.md

```diff
@@ -0,0 +1,80 @@
+
+Some of the limitations of the dns-module is as follows:
+* Although the SRV record name is provided in the variable, it is stored as `_service._proto.record_name.domain_name TTL class type of record priority weight port target`. For more information, see  [What is a DNS SRV record?](https://www.cloudflare.com/en-gb/learning/dns/dns-records/dns-srv-record/).
+
+
+* The changed name means that when you run a `terraform plan` command after a successful `terraform apply`, the output shows that the DNS record requires an update, as shown in the following example. You can ignore that message. Your infrastructure will not be affected.
+
+        # module.cis_dns_records[0].ibm_cis_dns_record.dns_records["NAME/SRV"] will be updated in-place
+        ~ resource "ibm_cis_dns_record" "dns_records" {
+            id          = "a5177ec049fc2973a33df1441e869a27:9684838a87ecxxx5518:crn:v1:bluemix:public:internet-svcs:global:a/abac0df06b644axxx4f55b3880e:6ee7ec9a-5e68-4b6f-af9a-5714xxx4d::"
+            ~ name      = "_sip._udp.test-example.srv.test**.**.com" -> "test-example.srv"
+            # (13 unchanged attributes hidden)
+            }
+
+* If you add a CAA record, a `flags` parameter is returned in the data object. The work is being tracked in `IBM-Cloud/terraform-provider-ibm` [issue 4792](https://github.com/IBM-Cloud/terraform-provider-ibm/issues/4792).
+
+* The returned `flags` parameter means that when you run a `terraform plan` command after a successful `terraform apply`, the output shows that the DNS record requires an update, as shown in the following example. You can ignore that message. Your infrastructure will not be affected.
+
+        # module.cis_dns_records.ibm_cis_dns_record.dns_records["test-exmple.caa/CAA"] will be updated in-place
+        ~ resource "ibm_cis_dns_record" "dns_records" {
+            ~ data        = {
+                - "flags" = "0" -> null
+                    # (2 unchanged elements hidden)
+                }
+                id        = "fcef7410xxxxxxbad23c5fd0e7581b7c:7e66xxxxxecc7e12ac908ca75445ad21:crn:v1:bluemix:public:internet-svcs:global:a/abac0df06b644axxxxx6e44f55b3880e:06240432-xxxx-40e7-9f9c-594dfbdfe208::"
+                # (12 unchanged attributes hidden)
+            }
+
+## Web Application Firewall(WAF)
+
+A Web Application Firewall (WAF) safeguards web applications by scrutinizing and overseeing HTTP traffic that flows between a web application and the internet. WAF is implemented through two rule sets: [OWASP](https://cloud.ibm.com/docs/cis?topic=cis-waf-settings#owasp-rule-set-for-waf) and [CIS](https://cloud.ibm.com/docs/cis?topic=cis-waf-settings#cis-ruleset-for-waf). For more information see, [Web Application Firewall concepts](https://cloud.ibm.com/docs/cis?topic=cis-waf-q-and-a).
+
+Currently, the [waf module](https://github.com/terraform-ibm-modules/terraform-ibm-cis/blob/main/modules/waf/) provides terraform resources to enable and disable WAF with default policies in a CIS instance.
```

Comment:
The [WAF submodule](https://github.com/terraform-ibm-modules/terraform-ibm-cis/blob/main/modules/waf/) provides the terraform resources to enable and disable WAF with the default policies (you can provide ruleset names here....what it shows on dashboard) in a CIS instance.

default policies -> check it out if word used is policies or rulesets.